packages/util/src/units.ts

```diff
@@ -0,0 +1,72 @@
+// Source of constants: https://www.languagesandnumbers.com/articles/en/ethereum-ether-units/
+export const unitsConstants = {
```

Comment:
Can you format this as a typescript enum `Unit`? I just tested and the value can be a negative number so that should work. Key should be PascalCase to match other enums in the monorepo. Then you shouldn't have to use any string values in the actual code, i.e. UnitName.Ether should === 'ether'